lib/parser.js

```diff
@@ -113,14 +113,28 @@ module.exports = function parser(identifier, string) {
 
     expect(Token.Type.OPERATOR) // equals
 
+    // Parses a singular value: string, number, bool, or a list of those.
+    var parseValue = function () {
+      var result
+      if (peek().type === Token.Type.START_OPTION) {
```

Comment:
How did you figure out that this is the correct syntax for repeated option values? I couldn't find a good example of this case.

Reply:
hmm, I don't know? i kinda assumed so and then i ran protoc on it

Reply:
for posterity

```python
from protos import kitchen_sink_pb2
from protos import options_pb2

print kitchen_sink_pb2.DESCRIPTOR.GetOptions().Extensions[options_pb2.surrogate_b].mapped

# > [u'url', u'entity']
```